src/Bicep.Core/CoreResources.resx

```diff
@@ -118,46 +118,46 @@
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
   <data name="BicepConfigCustomSettingsFoundFormatMessage" xml:space="preserve">
-    <value>Custom bicepconfig.json file found [{0}].</value>
+    <value>Custom bicepconfig.json file found ({0}).</value>
     <comment>message to indicate that a custom settings file was found {0} is the file location</comment>
   </data>
   <data name="BicepConfigNoCustomSettingsMessage" xml:space="preserve">
     <value>No bicepconfig.json found for configuration override.</value>
   </data>
   <data name="EnvironmentUrlHardcodedRuleDescription" xml:space="preserve">
-    <value>Environment URLs should not be hardcoded. Access URLs via the environment() function to keep references current.</value>
+    <value>Environment URLs should not be hardcoded. Use the environment() function to ensure compatibility across clouds.</value>
   </data>
   <data name="InterpolateNotConcatRuleDescription" xml:space="preserve">
-    <value>Dynamic variable should not use concat - string interpolation should be used.</value>
+    <value>Use string interpolation instead of the concat function.</value>
   </data>
   <data name="LinterDisabledFormatMessage" xml:space="preserve">
-    <value>Linter is disabled in settings file [{0}]</value>
+    <value>Linter is disabled in settings file ({0})</value>
     <comment>{0} is the location of the settings file</comment>
   </data>
   <data name="LinterRuleExceptionMessageFormat" xml:space="preserve">
     <value>Analyzer '{0}' encountered an unexpected exception. {1}</value>
     <comment>string format to display error exception thrown in Linter Rule</comment>
   </data>
   <data name="LocationSetByParameterRuleDescription" xml:space="preserve">
-    <value>Best practice dictates that location be through a parameter that has no default or defaults to 'global' or resourceGroup().location.</value>
+    <value>Resource location should be specified by a parameter with no default or that defaults to 'global' or resourceGroup().location.</value>
```

Comment:
>Resource location should be specified by a parameter with no default or that defaults to 'global' or resourceGroup().location. [](http://example.com/codeflow?start=11&length=126)

Can we rephrase to this:
`Resource location should be specified by a parameter with no default value or one that defaults to 'global' or resourceGroup().location.`